kubernetes/table_kubernetes_service.go

```diff
@@ -126,6 +126,12 @@ func tableKubernetesService(ctx context.Context) *plugin.Table {
 				Description: "A list of ports that are exposed by this service.",
 				Transform:   transform.FromField("Spec.Ports"),
 			},
+			{
+				Name:        "selector_query",
+				Type:        proto.ColumnType_STRING,
+				Description: "Route service traffic to pods with label keys and values matching this selector. String format representation.",
```

Comment:
This description should probably be consistent with the other `selector_query` descrptions